docs/README.md

```diff
@@ -0,0 +1,43 @@
+# Visual Beta Documentation
+
+## Intro
+
+Sauce Visual helps you to compare uploaded images (so called snapshots) against reference images (called baselines). It offers APIs to upload snapshots and compare those against baselines. The result of the comparison prrocess is a diff with details about eventually detected differences. All snapshots in Sauce Visual need to be assigned to visual builds.
```

Comment:
Typo: prrocess

Reply:
Also wondering if keywords that they will encounter on the UI might be in bold or code formatted - stuff like snapshot, baseline, diff

Reply:
It's interesting idea. Personally I would prefer to keep bold style for links to other resources.

Reply:
@aluedeke WYT?